src/init.cpp

```diff
@@ -358,6 +358,7 @@ std::string HelpMessage(HelpMessageMode mode)
     if (showDebug) {
         strUsage += HelpMessageOpt("-dbbatchsize", strprintf("Maximum database write batch size in bytes (default: %u)", nDefaultDbBatchSize));
     }
+    strUsage += HelpMessageOpt("-includeconf=<file>", _("Specify additional configuration file, relative to the -datadir path"));
```

Comment:
Help text could be expanded a bit to say that this can't be set from the command line - only from within the .conf file.

Reply:
I thought I had done that already. Fixing.

Reply:
Updated. Feedback welcome!